modules/contact/src/actions/ContactAction.C

```diff
@@ -200,6 +200,15 @@ ContactAction::validParams()
   params.addParam<std::vector<TagName>>(
       "extra_vector_tags",
       "The tag names for extra vectors that residual data should be saved into");
+  params.addParam<bool>(
+      "use_petrov_galerkin",
+      false,
+      "Whether to use the Petrov-Galerkin approach for the mortar-based constraints. If set to "
+      "true, we use the standard basis as the test function and dual basis as "
+      "the shape function for the interpolation of the Lagrange multiplier variable.");
+  params.addCoupledVar("aux_lm",
+                       "Auxiliary Lagrange multiplier variable that is utilized together with the "
+                       "Petrov-Galerkin approach.");
```

Comment:
It doesn't seem like you're using this parameter?

Reply:
That's true! Nice catch! 

It was in my original version which should be removed after switching to automatically adding an aux LM variable